docs/podman.1.md

```diff
@@ -182,13 +182,15 @@ the exit codes follow the `chroot` standard, see below:
 
 ## FILES
 
-**libpod.conf** (`/etc/containers/libpod.conf`)
+**libpod.conf** (`/usr/share/containers/libpod.conf`)
 
-    libpod.conf is the configuration file for all tools using libpod to manage containers.  When Podman runs in rootless mode, then the file `$HOME/.config/containers/libpod.conf` is used.
+    libpod.conf is the configuration file for all tools using libpod to manage containers, when run as root.  Administrators can override the defaults file by creating `/etc/containers/libpod.conf`.  When Podman runs in rootless mode, then the file `$HOME/.config/containers/libpod.conf` is created and overrides the system configuration file.
 
-**mounts.conf** (`/usr/share/containers/mounts.conf` and optionally `/etc/containers/mounts.conf`)
+**mounts.conf** (`/usr/share/containers/mounts.conf`)
```

Comment:
It's curious that the file /usr/share/containers/mounts.conf comes as part of the containers-common package, but the man page for it comes with the podman package. Should they not be part of the same package?

Reply:
Yes the man page should be in containers-common.  Since the mounts.conf is also used by buildah.

Reply:
Will fix this in separate PRs